### PR TITLE
Adding web files (html and css) and Linters

### PR DESCRIPTION
**Pull request by Oscarj-8**
Added HTML and CSS files with Linters and readme file updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
```
Comment (leonardodiasb) [inline index.html]: - [ ] You should import the CSS file in the `<head>` section of your HTML file in order to it apply the styling you added in your CSS file to your HTML elements. Kindly make sure you import the correct CSS file you created for this project and make sure the elements have the style you implemented.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Oscarj-8, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - `test.md` correctly ignored. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] Please describe in a professional way the changes you implemented in this Pull Request message and change the title to a descriptive one too. The Pull Request message should have a descriptive message of the features implemented to guide the developer who will review the code to understand it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] The branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). It would be nice if you change the name of the branch to make it easier to identify what is the purpose of this branch. Remember this for future projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hi @Oscarj-8  
### I'm Levy, your PR reviewer

 Excellent, you nailed it on your second attempt :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 



## Highlights
- Good job for using correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Oscarj-8** [comment]
so, can i merge it now?

    On Tuesday, September 20, 2022 at 05:27:44 PM GMT+3, levy_ukwishaka ***@***.***> wrote:  
 
 


@levy002 approved this pull request.

Hi @Oscarj-8

I'm Levy, your PR reviewer

Excellent, you nailed it on your second attempt 😎
Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉 🎉 🎉
 


Highlights
   
   - Good job for using correct git-flow ✔️
   - All linters are passing ✔️
   - Good commit history ✔️

Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Pull request merged**